packages/flutter_tools/lib/src/bundle.dart

```diff
@@ -85,7 +85,7 @@ Future<void> build({
 
     } else if (!fs.file(compilationTraceFilePath).existsSync()) {
       // Be forgiving if compilation trace file is missing.
-      printError('Warning: Ignoring missing compilation training file $compilationTraceFilePath.');
+      printError('No compilation trace available. To optimize performance, consider using --train.');
```

Comment:
it's not really an error, we can probably make it use printStatus. Otherwise people might get more concerned.